website/docs/tutorials/structured_config/1_minimal_example.md

```diff
@@ -83,5 +83,5 @@ It can be useful when you care about the methods or attributes of an object, not
 ## Overriding default values in the `@dataclass`
 You can use instances of the dataclasses to override default values in the stored config.
 ```python
-cs.store(name="config", node=MySQLConfig(user="root", password="1234"))
+cs.store(name="config", node=MySQLConfig(host="anotherhost", port=80))
```

Comment:
why this change?

Reply:
This example has no `user` / `password` fields => they trigger an error

Reply:
Oh, I see.
```suggestion
cs.store(name="config", node=MySQLConfig(host="test.db", port=3307))
```

Reply:
Taking your change, but FYI I had intentionally used port 80 because it triggered something "interesting" in the output